ui/components/app/snaps/insight-warnings/insight-warnings.js

```diff
@@ -149,12 +156,14 @@ export default function InsightWarnings({
           }}
         >
           <Checkbox
-            variant={TextVariant.bodySm}
+            variant={TextVariant.bodyMd}
             isChecked={isChecked}
             onChange={handleOnChange}
             label={t('insightWarningCheckboxMessage', [
               t(InsightWarningLanguage[action].imperative),
-              stripHttpSchemes(origin),
+              <Text as="span" key={action} fontWeight={FontWeight.Bold}>
```

Comment:
```suggestion
              <Text as="span" key={action} fontWeight={FontWeight.Medium}>
```
I believe we usually use Medium for these

Reply:
Fixed, although I think the difference is too little to bring awareness. But again, it's up to @eriknson 

![Screenshot 2024-03-07 at 10 56 59](https://github.com/MetaMask/metamask-extension/assets/44679989/325175b8-e64a-493f-b144-f823d3636a8d)
![Screenshot 2024-03-07 at 10 56 23](https://github.com/MetaMask/metamask-extension/assets/44679989/3b97ba10-7c58-48a8-bc2b-18393be52071)